### DISCUSSION TITLE
Origin app

### DISCUSSION THREAD

**Discussion by dfhdrfhrdehrdhrdfh** [PhilipK/BoilR#392] [Ideas]
Hello, thanks for this great tool.

1 little suggestion. 
For Origin games BoilR looks for the /origins folder.
Now EA moved away from the origin app in favour of their new EA app. 

All games can now be found in the /EA Desktop folder.

Thank!

**Comment by Fury-hub** [comment]
Yes origin should be replaced by EA app
